apps/studio/components/interfaces/Settings/Database/NetworkRestrictions/NetworkRestrictions.tsx

```diff
@@ -101,187 +109,164 @@ const NetworkRestrictions = () => {
-                  </p>
-                </div>
-                <div className="flex items-center space-x-2">
-                  <AllowAllAccessButton
-                    disabled={!canUpdateNetworkRestrictions}
-                    onClick={setIsAllowingAll}
-                  />
-                  <DisallowAllAccessButton
-                    disabled={!canUpdateNetworkRestrictions}
-                    onClick={setIsDisallowingAll}
-                  />
-                </div>
-              </div>
-            </Panel.Content>
-          </Panel>
-        ) : (
-          <FormPanel>
-            {isUninitialized || isAllowedAll ? (
-              <div className="px-8 py-8 flex items-center justify-between">
-                <div className="flex items-start space-x-4">
-                  <div className="space-y-1">
-                    <p className="text-foreground-light text-sm">
-                      Your database can be accessed by all IP addresses
-                    </p>
-                    <p className="text-foreground-light text-sm">
-                      You may start limiting access to your database by adding a network
-                      restriction.
-                    </p>
-                  </div>
-                </div>
-                <div>
-                  <DisallowAllAccessButton
-                    disabled={!canUpdateNetworkRestrictions}
-                    onClick={setIsDisallowingAll}
-                  />
-                </div>
-              </div>
-            ) : isDisallowedAll ? (
-              <div className="px-8 py-8 flex items-center justify-between">
-                <div className="flex items-start space-x-4">
-                  <Lock size={20} className="text-foreground-light" strokeWidth={1.5} />
-                  <div className="space-y-1">
-                    <p className="text-foreground-light text-sm">
-                      Your database <span className="text-amber-900 opacity-80">cannot</span> be
-                      accessed externally
-                    </p>
-                    <p className="text-foreground-light text-sm">
+          <Admonition type="warning" title="Your network restrictions were not applied correctly">
+            <p>Please try to add your network restrictions again</p>
+            <div className="flex gap-2">
+              <AllowAllAccessButton
+                disabled={!canUpdateNetworkRestrictions}
+                onClick={setIsAllowingAll}
+              />
+              <DisallowAllAccessButton
+                disabled={!canUpdateNetworkRestrictions}
+                onClick={setIsDisallowingAll}
+              />
+            </div>
+          </Admonition>
+        ) : isUninitialized || isAllowedAll ? (
+          <Card>
+            <CardContent className="flex gap-4">
+              <Unlock />
+              <FormLayout
+                layout="flex-row-reverse"
+                label="Any IP address can access your database"
```

Comment:
changed the wording on this title so that it didn't wrap at smaller screen sizes